### PR TITLE
feat: migrate VCent plugin to new plugin infra

### PR DESCRIPTION
**Pull request by UncleBill**
part of #3066

### CODE REVIEW

File: packages/maskbook/src/plugins/VCent/base.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { PLUGIN_ID, PLUGIN_ICON, PLUGIN_NAME, PLUGIN_DESCRIPTION } from './constants'
+
+export const base: Plugin.Shared.Definition = {
+    ID: PLUGIN_ID,
+    icon: PLUGIN_ICON,
+    name: { fallback: PLUGIN_NAME },
+    description: { fallback: PLUGIN_DESCRIPTION },
+    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/VCent/base.ts:9]: IIRC this plugin is contributed by the community right?

File: packages/maskbook/src/plugins/VCent/constants.ts [hunk 2]
```diff
@@ -4,3 +4,9 @@ export const PLUGIN_IDENTIFIER = 'com.maskbook.tweet'
 export const TWEET_BASE_URL = 'https://vcent-agent.r2d2.to/data/tweet-txn?tweetID='
 export const VALUABLES_VCENT_URL = 'https://v.cent.co/tweet/'
 //#endregion
+
+export const PLUGIN_META_KEY = 'com.maskbook.tweet:1'
+export const PLUGIN_ID = 'com.maskbook.tweet'
+export const PLUGIN_NAME = 'Gitcoin'
```
Comment (guanbinrui) [inline packages/maskbook/src/plugins/VCent/constants.ts:10]:
```suggestion
export const PLUGIN_NAME = 'vCent'
```

File: packages/maskbook/src/plugins/VCent/base.ts (hunk 1 above)
Comment (guanbinrui) [reply packages/maskbook/src/plugins/VCent/base.ts:9]: @yisiliu Can you provide the provider's information?

Comment (Jack-Works) [reply packages/maskbook/src/plugins/VCent/base.ts:9]: it was https://github.com/DimensionDev/Maskbook/pull/2683

Comment (guanbinrui) [reply packages/maskbook/src/plugins/VCent/base.ts:9]: @UncleBill Take a look?

Comment (UncleBill) [reply packages/maskbook/src/plugins/VCent/base.ts:9]: @guanbinrui I will update later.

File: packages/maskbook/src/plugins/PluginUI.ts [hunk 3]
```diff
@@ -16,8 +16,6 @@ import { DHedgePluginDefine } from './dHEDGE/define'
 
 sideEffect.then(() => {
     if (Flags.ito_enabled) plugins.add(ITO_PluginDefine)
-    if (Flags.gitcoin_enabled) plugins.add(GitcoinPluginDefine)
-    if (Flags.poll_enabled) plugins.add(PollsPluginDefine)
```
Comment (septs) [inline packages/maskbook/src/plugins/PluginUI.ts]: @Jack-Works is need delete unused flags?

Comment (Jack-Works) [reply packages/maskbook/src/plugins/PluginUI.ts]: yes

File: packages/maskbook/src/plugins/VCent/base.ts (hunk 1 above)
Comment (UncleBill) [reply packages/maskbook/src/plugins/VCent/base.ts]: Updated the publisher as @rob-lw

**Review by Jack-Works — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**